mlflow/server/js/src/utils/Utils.js

```diff
@@ -162,12 +162,25 @@ class Utils {
     return url;
   }
 
+  /**
+   * Returns a copy of the provided URL with its query parameters set to `queryParams`.
+   * @param url URL string like "http://my-mlflow-server.com/#/experiments/9.
+   * @param queryParams Optional query parameter string like "?param=12345". Query params provided
+   *        via this string will override existing query param values in `url`
+   */
+  static setQueryParams(url, queryParams) {
+    const urlObj = new URL(url);
+    urlObj.search = queryParams || "";
```

Comment:
Changed this (from #968) to just set the query params instead of adding `queryParams` to the existing ones in the passed-in URL, which removed the need to use the `query-string` library & URLSearchParams interface